src/conways.rs

```diff
@@ -39,14 +41,14 @@ impl ConwaysGrid {
         let previous_grid = self.grid.clone();
         for row in 0..ROWS {
             for col in 0..COLS {
-                self.compute_next_state(&previous_grid, (row, col));
+                let alive =
+                    Self::get_alive_count(&previous_grid, self.get_neighbor_position((row, col)));
```

Comment:
Keeping `previous_gris` as a `ConwaysGrid` would make this simpler. Then you could juat make this:


```suggestion
                    self.get_alive_count(row, col);
```

Reply:
Thanks for the suggestion, addressed in #6.